cmp/grid/GridModel.js

```diff
@@ -115,6 +116,8 @@ export class GridModel {
     /**
      * @param {Object} c - GridModel configuration.
      * @param {Object[]} c.columns - {@link Column} or {@link ColumnGroup} configs
+     * @param {Object} [c.defaultCol] = Column configs to be set on all columns.  If a Column has
```

Comment:
I feel that this comment states the obvious regarding what "wins out".  That's fine, but  I think the less obvious point that we should clarify is that the merging is done "deeply"...  I would not assume that as an API user and would be glad to have it clarified....